// 自定义错误地址
// 当错误发生的时候，我们捕获到发生的错误，然后抛出我们自定义的错误
